src/components/ha-button-menu.ts

```diff
@@ -50,6 +51,24 @@ export class HaButtonMenu extends LitElement {
     `;
   }
 
+  protected firstUpdated(changedProps): void {
+    super.firstUpdated(changedProps);
+
+    document.querySelector("home-assistant")!.provideHass(this);
```

Comment:
Can we use `document.body.dir` instead of querying `hass`?

Reply:
Done